bundles/org.openhab.binding.surepetcare/src/main/resources/ESH-INF/thing/things.xml

```diff
@@ -0,0 +1,619 @@
+
+		<label>Sure Petcare Hub Device</label>
+		<description>A Sure Petcare hub device (connects cat flaps, feeders etc. to a network)</description>
+
+		<channels>
+			<channel id="id" typeId="idType" />
+			<channel id="name" typeId="nameType" />
+			<channel id="product" typeId="productType" />
+			<channel id="ledMode" typeId="ledModeType" />
+			<channel id="pairingMode" typeId="pairingModeType" />
+			<channel id="online" typeId="onlineType" />
+		</channels>
+
+		<properties>
+			<property name="id" />
+			<property name="version" />
+			<property name="createdAt" />
+			<property name="updatedAt" />
+			<property name="householdId" />
+			<property name="productType" />
+			<property name="productName" />
+		</properties>
+
+	</thing-type>
+
+	<thing-type id="flapDevice">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Sure Petcare Flap Device</label>
+		<description>A Sure Petcare cat or pet flap device</description>
+
+		<channels>
+			<channel id="id" typeId="idType" />
+			<channel id="name" typeId="nameType" />
+			<channel id="product" typeId="productType" />
+			<channel id="curfewEnabled1" typeId="curfewEnabledType" />
+			<channel id="curfewLockTime1" typeId="curfewLockTimeType" />
+			<channel id="curfewUnlockTime1" typeId="curfewUnlockTimeType" />
+			<channel id="curfewEnabled2" typeId="curfewEnabledType" />
+			<channel id="curfewLockTime2" typeId="curfewLockTimeType" />
+			<channel id="curfewUnlockTime2" typeId="curfewUnlockTimeType" />
+			<channel id="curfewEnabled3" typeId="curfewEnabledType" />
+			<channel id="curfewLockTime3" typeId="curfewLockTimeType" />
+			<channel id="curfewUnlockTime3" typeId="curfewUnlockTimeType" />
+			<channel id="curfewEnabled4" typeId="curfewEnabledType" />
+			<channel id="curfewLockTime4" typeId="curfewLockTimeType" />
+			<channel id="curfewUnlockTime4" typeId="curfewUnlockTimeType" />
+			<channel id="lockingMode" typeId="lockingModeType" />
+			<channel id="lowBattery" typeId="system.low-battery" />
+			<channel id="batteryLevel" typeId="system.battery-level" />
+			<channel id="batteryVoltage" typeId="batteryVoltageType" />
+			<channel id="online" typeId="onlineType" />
+			<channel id="deviceRSSI" typeId="rssiDeviceType" />
+			<channel id="hubRSSI" typeId="rssiHubType" />
+		</channels>
+
+		<properties>
+			<property name="id" />
+			<property name="version" />
+			<property name="createdAt" />
+			<property name="updatedAt" />
+			<property name="householdId" />
+			<property name="productType" />
+			<property name="productName" />
+			<property name="pairingAt" />
+		</properties>
+	</thing-type>
+
+	<thing-type id="feederDevice">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Sure Petcare Feeder Device</label>
+		<description>A Sure Petcare pet feeder device</description>
+
+		<channels>
+			<channel id="id" typeId="idType" />
```

Comment:
Please check the channels you expose, it seems you expose a lot, consider marking some (more) advanced:

> Especially for complex devices with a lot of channels, only a small set of channels - the most important ones - should be shown to the user to reduce complexity. Whether a channel should be declared as advanced depends on the device and can be decided by the binding developer. If a functionality is rarely used it should be better marked as advanced.